app/components/blobs/BlobWithText.tsx

```diff
@@ -0,0 +1,64 @@
+interface BlobWithTextProps {
+  text: string;
+  blobPathNr: number;
+}
+
+export const BlobWithText = (props: BlobWithTextProps) => {
+  const { text, blobPathNr } = props;
+
+  const paths = [
+    "M212.326 383.801C269.807 387.285 320.144 344.505 352.507 299.58C379.97 261.457 365.853 213.445 368.207 167.413C370.782 117.032 403.344 58.6133 366.704 21.8604C329.965 -14.9919 265.506 4.42725 212.326 11.0525C168.518 16.51 128.806 30.1895 93.9825 55.8361C52.7163 86.2276 1.07778 118.053 0.0155288 167.413C-1.05338 217.082 53.2994 247.144 89.0807 283.612C127.373 322.64 156.052 380.39 212.326 383.801Z",
+    "M219.356 383.847C257.585 385.842 293.271 368.091 322.149 342.253C353.51 314.192 383.293 279.284 383.989 236.541C384.693 193.337 350.278 162.236 325.651 127.227C292.896 80.6651 274.102 14.0374 219.356 2.5061C160.202 -9.95349 99.791 25.7992 58.3026 70.9153C17.8459 114.909 -11.1308 178.061 4.1394 236.541C17.9435 289.405 82.179 301.716 126.489 332.044C156.965 352.904 182.793 381.94 219.356 383.847Z",
+    "M198.044 383.418C258.85 376.942 283.593 314.371 314.61 268.83C335.722 237.834 336.674 203.661 344.519 168.36C355.838 117.421 408.952 59.7016 369.84 20.286C331.147 -18.7085 255.798 8.30243 198.044 20.7458C156.495 29.6976 127.192 55.2039 94.9596 79.5101C58.8107 106.77 10.9676 127.641 2.29616 168.36C-7.49116 214.318 15.1469 260.397 48.609 297.16C87.0768 339.422 136.192 390.005 198.044 383.418Z",
+    "M166.726 383.968C212.957 385.164 249.793 352.88 285.429 323.814C326.626 290.213 385.059 259.472 383.985 206.763C382.915 154.215 319.394 130.525 280.756 94.3206C243.886 59.7735 216.933 11.2101 166.726 3.19284C110.174 -5.83768 38.3284 2.55884 8.62839 50.8658C-20.3832 98.053 32.2998 152.343 41.5673 206.763C48.0708 244.953 31.6472 286.054 54.0911 317.831C80.0529 354.588 121.315 382.793 166.726 383.968Z",
+    "M187.242 383.863C238.136 386.061 286.426 361.563 322.747 324.108C359.392 286.322 389.953 235.394 383.009 181.992C376.52 132.076 326.339 107.2 290.227 73.9825C257.974 44.3157 229.758 7.92822 187.242 1.92139C139.186 -4.86823 87.9175 5.96573 52.1915 40.3516C15.2411 75.916 -1.82495 129.434 0.154124 181.992C2.06136 232.642 28.7666 276.521 62.4532 312.869C96.7413 349.867 138.089 381.741 187.242 383.863Z",
+  ];
+
+  return (
+    <div
+      className="flex justify-center text-[#028377] hover:text-[#01574F]"
```

Comment:
ikke helt frelst av komboen med denne bakgrunnen og bakgrunnen på selve appen, men trenger ikke å bestemme noe av det nå da. En designer kan komme med pekefingeren sin 😁

Reply:
Forslag til farger du ønsker å bruke? Jeg kunne jo gjort den generisk mtp farger også.

Reply:
Ja, kanskje sende inn fargekode som props? evt globale konstanter? Synes denne er ganske bra for å lage fargepaletter: https://coolors.co/